v20_0_prerelease/location.js

```diff
@@ -329,6 +340,25 @@ class FormItMap {
         this._deselectStation();
     }
 
+    useLastLocation() {
```

Comment:
Naming of this is confusing. This uses the location from the file, so I'd name it that way. Perhaps syncLocationFromFile